### DISCUSSION TITLE
Ability to add add bubble maps of HAB Red Tide Cell Counts to map

### DISCUSSION THREAD

**Discussion by Philliec459** [opengeos/HyperCoast#31] [Q&A]
Hello, we are new to HyperCoast and affiliated projects, but would like to add our time series data for Hazardous Algal Blooms (HAB) to our PACE reflectance maps as bubble maps. With summer here in Florida, we would like to start routinely downloading PACE data to look for relative changes in PACE data throughout the summer using HyperCoast. We have access to time series Red Tide Cell Count data by lat/lon (shown below) that we would like to integrate with the PACE data . We will also do the same for the Lake Okeechobee area too. Lake O is known for near-continuous HAB’s, and again looking for relative changes in our PACE data if PACE is applicable for fresh waters too. 

Also, your HyperCoast PACE example is great and your example data covers our region very well.  Is there a system to downloading PACE data over this same region to create a number of time series examples to show relative changes that we might be able to relate to HABs? 

![Red_Tide_SW_Florida_by_date](https://github.com/opengeos/HyperCoast/assets/56929340/56c27318-28cb-4f20-9ac9-2f3dd16dd688)

**Comment by giswqs** [comment]
Are the lat/lon data publicly available? I can add another example for adding the bubble data.

**Reply by Philliec459** [reply]
That would be great. Thank you. Dr. Dave Tomasko of Sarasota Bay Estuary Program sent us this source for Harmful Algal Bloom data from NOAA. 

https://habsos.noaa.gov

We have been using an ipywidget range slider to create our bubble map for only the date range that we are using. We also used stumpy to compare time series patterns.

![Red_Tide_SW_Florida_Patterns_stumpy](https://github.com/opengeos/HyperCoast/assets/56929340/797160a9-d097-49bb-81b0-2c3ea5333e5a)

**Reply by giswqs** [reply]
Can you share any notebook examples or Python scripts that already developed for this use case? It will make it easier for me to integrate the features into HyperCoast

**Reply by Philliec459** [reply]
Please find attached our very rough python code with NOAA HAB Red Tide Cell Counts for the Sarasota Bay area that we used to create the plot below. This was meant to go into a repository on GitHub, but we got sidetracked with our Landsat interpretations of NDVI where we are attempting to differentiate Mangrove Habitats and species over an area to the west of Bradenton, FL called Robinson Preserve(https://github.com/Philliec459/Science-and-Technology-Society-Use-of-NASA-Landsat-Data-to-Calculate-NDVI-and-PNDVI). 
[HAB_Interactive_Plots_with_Stumpy.zip](https://github.com/opengeos/HyperCoast/files/15349643/HAB_Interactive_Plots_with_Stumpy.zip)
